packages/cli/src/utils/environment-tasks.ts

```diff
@@ -44,25 +44,17 @@ const waitToBeReady = async (endpoint?: string): Promise<void> => {
 
 export type IStop = { configDir: string }
 export const stop: ListrTask<IStop> = {
-  title: 'Stop environment',
+  title: 'Stopping environment',
   task: () => new Listr([
     {
       title: 'Stopping the Engine',
       skip: ctx => hasOtherInstances(ctx.configDir),
       task: async () => {
-        const services = await listServices({ name: ['engine'] })
-        if (services.length === 0) throw new Error('Cannot find engine')
-        const service = services[0]
-        let image = (service.data as any).Spec.TaskTemplate.ContainerSpec.Image
-        image = [
-          image.split(':')[0],
-          image.split(':')[1] || 'latest'
-        ].join(':')
-        const eventPromise = waitForEvent(({ Action, from, Type }) => {
-          return Type === 'container' && Action === 'destroy' && from === image
-        })
-        await service.remove()
-        await eventPromise
+        const containers = await listContainers({ label: ['mesg.engine=true'] })
+        if (containers.length === 0) throw new Error('Cannot find engine')
+        const container = containers[0]
+        await container.stop()
+        await container.delete()
```

Comment:
will be nice to also delete the engine's network

Reply:
done